recipes-app/switchserialmode/files/src/switchserialmode.c

```diff
@@ -412,7 +414,7 @@ static void ttyuart_command_handle(int argc, char **argv)
     }
     else if(0 == strcasecmp(mode, "rs422"))
     {
-        ttyuart_switchto_rs422(devType);
+        ttyuart_switchto_rs422(devType, logicLevel);
```

Comment:
This file was probably committed by accident.

Reply:
No, it's on purpose. The `SER_RS485_RX_DURING_TX` is used to tell apart the RS422 from RS485. Also the other change is needed to let the RS422 fully functioning.

Actually this file is far away from what it should be, I was working on a refactory, however, at this moment maybe just to fix the issue is ok.

Reply:
It's not ok to merge different topics into the same commit.

Reply:
ACK w/ revision, please review